### ISSUE TITLE
output DAG size and build time

### ISSUE THREAD

**Issue by nerdralph** [ethereum-mining/ethminer#566]
With on-GPU DAG generation, the DAG load progress should be removed and replaced as I did with ethminer-nr.
"GPU1  2.27GB of DAG data generated in 3885 ms."
https://github.com/nerdralph/ethminer-nr/blob/master/libethash-cl/ethash_cl_miner.cpp#L483

**Comment by nerdralph** [comment]
I know the DAG is generated on the GPU.  Showing the percentage progress is of little benefit since it only takes a few seconds.
I provided a link to my code, since there are so many changes since I implemented it over a year ago that it's not a simple diff to merge.

**Comment by ddobreff** [comment]
Its already done in latest git release.
```
[2018-01-15 20:59:47][info][miner]:  cu  20:59:47|CUDA2     Generating mining buffers
[2018-01-15 20:59:47][info][miner]:  cu└╧  20:547|CUDA6     Allocating light with size: 38141888
[2018-01-15 20:59:47][info][miner]:  cu└╧  20:547|CUDA1     Generating mining buffers
[2018-01-15 20:59:47][info][miner]:  cu└╧  20:547|CUDA5     Generating DAG for GPU # 5  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff62c000960
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:54:37|CU2     Generating DAG for GPU # 2  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff63c000960
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:54:37|CU3     Generating mining buffers
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA7     Generating mining buffers
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA4     Generating DAG for GPU # 4  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff628000960
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA6     Generating mining buffers
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA3     Generating DAG for GPU # 3  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff630000a70
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA1     Generating DAG for GPU # 1  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff638000b30
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:54837|CUDA7   Generating DAG for GPU # 7  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff624000960
[2018-01-15 20:59:48][info][miner]:  cu└╧  20:548|CUDA6     Generating DAG for GPU # 6  with dagSize: 2441084288  gridSize: 4096  &m_streams[0]: 0x7ff620000960
[2018-01-15 20:59:54][info][core]: Sending stats to server...
[2018-01-15 20:59:59][info][miner]:   Б└╧  20:55937|stratum   Received new jobb9f5b0243e  seed: #ec6e8e0d8056cf9eb61046eaeb754b8e  target: #0000000112e0be826d694b2e
[2018-01-15 20:59:59][info][miner]:   Б└╧20:59:59|stratum   Received new job #7af31371  seed: #ec6e8e0d8056cf9eb61046eaeb754b8e  target: #0000000112e0be826d694b2e
[2018-01-15 20:59:59][info][miner]:   m└╧  20:559|ethminer  Speed   0.00 Mh/s    gpu/0  0.00 54C 40%  gpu/1  0.00 46C 40%  gpu/2  0.00 53C 40%  gpu/3  0.00 54C 40%  gpu/4  0.00 54C 40%  gpu/5  0.00 54C 40%  gpu/6  0.00 52C 40%  gpu/7  0$
[2018-01-15 21:00:00][info][miner]:   Б└╧ 1:00:0:40|CUDA1   Solution found; Submitted to eu2.ethermine.org
[2018-01-15 21:00:00][info][miner]:   Б└╧ 1:00:0:40|CU1     Nonce: 0x01ceba8056f7b947
[2018-01-15 21:00:00][info][miner]:   Б└╧ 1:00:00|stratum    Accepted.
```

**Comment by jean-m-cyr** [comment]
> Showing the percentage progress is of little benefit since it only takes a few seconds.

We don't do that anymore, regardless of how long it takes.

**Comment by nerdralph** [comment]
@jean-m-cyr Just one line to delete.
https://github.com/ethereum-mining/ethminer/blob/master/libethash-cl/CLMiner.cpp#L727

**Comment by jean-m-cyr** [comment]
Yeah, I know. But, I try to focus on cuda since cl is not my thing. I don't know how fond the cl crowd is of their startup noise. Seems some are fussy about their console output! I got a few negative revues when I eliminated all the redundant (identical) per GPU 'got work' messages with every new job arrival in cuda.

I don't see any value in it other than maybe when debugging. I'll push a request up the pole and see who salutes.
